l10n_it_fatturapa_pec/models/fetchmail.py

```diff
@@ -158,26 +135,60 @@ def fetch_mail(self):
             server.write({'date': fields.Datetime.now()})
         return True
 
+    @api.multi
+    def manage_pec_failure(self, exception, error_messages):
```

Comment:
Riesci mica ad aggiungere un test?

Reply:
Ci avevo pensato ma non ho una mail da cui partire quindi avevo lasciato perdere.
Ora che mi ci fai ripensare però, andrebbe bene partire dal file `POSTA CERTIFICATA: Invio File 7339338.txt` e introdurre un errore nell'allegato `IT02652600210_00007`?

Reply:
Tipo che errore?

Reply:
Ho aggiunto un test, però verifica solo che venga sollevato un errore durante il processamento della mail (e in effetti è indipendente dalle mie modifiche).

Il vero test sarebbe verificare che, in seguito a un errore, venga inviata una mail ai `contacts_to_notify`, ma per fare questo dovrei simulare un mail server e la faccenda si fa complicata.
Se ho tempo mi ci metto

Reply:
> Tipo che errore?

Una qualsiasi errore che faccia fallire il processamento del file XML sollevando un'eccezione.

Ho aggiunto un test che verifichi l'invio della mail, vedi se va bene? Grazie

Reply:
> Una qualsiasi errore che faccia fallire il processamento del file XML sollevando un'eccezione

Chiedevo un esempio reale

Reply:
> > Una qualsiasi errore che faccia fallire il processamento del file XML sollevando un'eccezione
> 
> Chiedevo un esempio reale

A un nostro cliente è capitato un XML che solleva questa eccezione (se importato manualmente):
![image](https://user-images.githubusercontent.com/32064796/64678406-cacbdc80-d479-11e9-82de-380134d82599.png)

Mentre la mail contenente il file era stata solo segnata come letta.